### DISCUSSION TITLE
self hosted page with help guide (without login)

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#10318] [Help] [answered]
Hi, 
i am running a self hosted version. The default page goes to a login page. is it possible to set default page to a general page which offers some information/link to login page?

**Comment by andz-bb** [comment, marked as answer]
Hey @sghriyadh, with the latest Budibase release we have introduced custom branding for customers on the Pro plan or higher. This includes the ability to customise text on the header and login button of the login page. While it's not possible to change the default page from the login page you could potentially add some information to that page. Would it be useful for you to be able to customise more content on the login page? If so, I can request it as an enhancement.

**Comment by sghriyadh** [comment]
with so many apps possible under one account for many users. There should be a central landing page with public pages/announcement/wiki etc.
